lib/rucio/daemons/bb8/bb8.py

```diff
@@ -153,10 +155,10 @@ def run_once(
                 )
                 rse["secondary"] = usage_dict["expired"]["used"]
                 rse["total"] = (
-                    usage_dict["storage"]["total"]
-                    - usage_dict["min_free_space"]["used"]
+                    usage_dict["static"]["total"]
+                    - get_rse_limits(rse_id=rse["id"])["MinFreeSpace"]
```

Comment:
We need to make this an option too, somehow. I think we should all talk about this.